### PR TITLE
TTS GitHub Repo

### PR DESCRIPTION
**Pull request by its-a-lisa-at-work**
github repos that TTS Tech Portfolio manages and owns

### CODE REVIEW

**Comment by its-a-lisa-at-work** [comment]
@hoaiandang the active GitHub repos are the ones that should be added for https://github.com/18F/tts-tech-portfolio/issues/527

File: how_we_work/github.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+## TTS Tech Portfolio
+### GitHub Repos
+[18F tts-tech-portfolio](https://github.com/orgs/18F/teams/tts-tech-portfolio/repositories)
```
Comment (afeld) [inline how_we_work/github.md:3]: Given that they are already listed:

- At the link above
- [In Terraform](https://github.com/18F/tts-tech-portfolio/blob/master/terraform/main.tf)
- [Linked to the project board](https://github.com/orgs/18F/projects/11/settings/linked_repositories)

is there an advantage to listing them yet another place?

Comment (its-a-lisa-at-work) [reply how_we_work/github.md:3]: Yes, you have to go looking to find them there and if you're not involved in either of those activities, you would never find them.

Comment (its-a-lisa-at-work) [reply how_we_work/github.md:3]: @afeld nudge

**Comment by its-a-lisa-at-work** [comment]
Would this pull request benefit from having a separate issue attached that can be groomed so the user story is documented and folks can agree on acceptance criteria?

File: how_we_work/github.md (hunk 1 above)
Comment (afeld) [reply how_we_work/github.md:3]: Right; could we keep the line above without the table below?

Comment (its-a-lisa-at-work) [reply how_we_work/github.md:3]: only keeping the line above w/o the table means that you have to be a member of 18F Org to see which repositories the TTS Tech Portfolio has -- so it'd be a less transparent way of doing things and there might be some edge cases that would take extra work to give folks the information. 

The original use-case of letting Hoaian know which repos we own is OBE at this point -- so I feel like it is a matter of preference. 

I feel on a scale of 1-10; a 6.5 that I'd like to keep the table. So if you feel stronger about only keeping the link, than we can go in that direction instead.

**Pull request merged**